### DISCUSSION TITLE
Catch Rate and Facility Level Spawning

### DISCUSSION THREAD

**Discussion by Geodude6** [Deijin27/RanseiLink#56] [Show and tell]
I have found a correlation between a Pokemon's catch rate and the structure level required for that Pokemon to spawn in a location. 

* Facility Level 1: Catch Rate 180-255
* Facility Level 2: Catch Rate 100-170
* Facility Level 3: Catch Rate 3-90

These are the tests I performed:

* Set the catch rate of Starly, Meowth, Bidoof, and Minccino (normally appear at FL 1) to 90. This caused all of these Pokemon to stop spawning in level 1 wild areas, with nothing at all spawning in the Aurora farm and only Wooper spawning in the Aurora ravine. I repeated the test for 12 months and this was consistent.
* Set the catch rate of Charmander, Zubat, Tepig (normally appear at FL 1), Charmeleon, and Pignite (normally appear at FL 2) to 90; this caused no Pokemon to spawn in Ignis cave level 1. Upgraded Ignis cave to level 2, and only Darumaka spawned. I repeated the test for 12 months and this was consistent.
* Set the catch rate of Snorlax (normally appears at FL 3) to 255. Snorlax began spawning in the Aurora ravine level 1. I did not repeat this test as Snorlax appeared on the first attempt.

![Pokemon Conquest - Copy__32461](https://github.com/Deijin27/RanseiLink/assets/26030289/54e452ea-3985-4313-904a-be81d7658a35)

Afaict this was previously unknown so I hope this helps 🐸🙏

**Comment by Deijin27** [comment]
This is a really cool find.

This makes me think the "habitat" values are just a reverse lookup to make it faster to display where a pokemon appears in the gallery thing.

So if the building level is determined by the pokemons "catch rate" (named as such because Baconfry noticed that they were the same numbers as the catch rate from the main games), then it's just what decides whether a pokemon appears in a kingdom that remains to be discovered.

**Reply by Geodude6** [reply]
I do think calling it "catch rate" is still accurate, lower values do make the Pokemon harder to link with (ie, the link minigame moves faster). I also have a suspicion that catch rate also controls how frequently a Pokemon spawns, but I have not done any testing on this theory at this time.
